src/main/java/minijava/ir/optimize/AliasAnalyzer.java

```diff
@@ -824,6 +837,7 @@ public boolean transform() {
       Node ptr = sideEffect.getPred(1);
       Set<IndirectAccess> aliasClass = getPointsTo(ptr);
       boolean isLoad = sideEffect.getOpCode() == iro_Load;
+      assert !aliasClass.isEmpty() : "Empty alias class at " + sideEffect;
```

Comment:
We could check a environment variable (e.g. `MJ_ASSERTIONS`) for enabling assertions in the `run` script and set the environment variable by default in all testing scenarios.